pilot/proxy/envoy/testdata/cds-circuit-breaker.json.golden

```diff
@@ -143,6 +143,13 @@
     }
    },
    {
+    "name": "out.aeff8e226c5af0341357dfb68f238d8120355615",
+    "service_name": "172.217.0.0/16|external-TCP-443",
```

Comment:
is / a valid character in cluster name?

Reply:
It seems Envoy does not care about service name characters, it works.